css/amp.css

```diff
@@ -334,3 +334,13 @@ amp-analytics {
   overflow: hidden !important;
   visibility: hidden;
 }
+
+amp-user-notification {
+  position: fixed !important;
```

Comment:
@dvoytenko @cramforce i recorded a gif below showing the element taking us space. (this happens 1/50 refreshes)
you guy happen to see what I'm doing wrong here?

![jump-around](https://cloud.githubusercontent.com/assets/354746/11943173/97bcd2e0-a7f0-11e5-93f2-ccde5f91e243.gif)